metrics/sinks/influxdb/influxdb.go

```diff
@@ -82,12 +82,17 @@ func (sink *influxdbSink) ExportData(dataBatch *core.DataBatch) {
 
 			point := influxdb.Point{
 				Measurement: measurementName,
-				Tags:        metricSet.Labels,
+				Tags:        make(map[string]string),
```

Comment:
Commit 4436e46 on watercraft/heapster is intended to address this feedback.
Is there more I need to do?